app/integration/runtime/pom.xml

```diff
@@ -167,6 +166,7 @@
             <ignoredUsedUndeclaredDependency>org.springframework:spring-beans</ignoredUsedUndeclaredDependency>
             <ignoredUnusedDeclaredDependency>org.apache.camel:camel-spring</ignoredUnusedDeclaredDependency>
             <ignoredUnusedDeclaredDependency>javax.xml.bind:jaxb-api</ignoredUnusedDeclaredDependency>
+            <ignoredUnusedDeclaredDependency>com.google.code.findbugs:findbugs-annotations</ignoredUnusedDeclaredDependency>
```

Comment:
perhaps we can remove the dependency declaration instead of adding the ignoredUnusedDeclaredDependency?

Reply:
The dependency comes from core/model which needs to suppress some [findbug warning](https://github.com/syndesisio/syndesis/blob/master/app/core/model/src/main/java/io/syndesis/model/WithConfigurationProperties.java#L33) on immutables generated code. If you know how to get rid of such dependency I'm more than happy.